zigpy/application/state.py

```diff
@@ -0,0 +1,220 @@
+"""Classes to implement status of the application controller."""
+
+from dataclasses import InitVar, dataclass, field
+import functools
+from typing import Any, Dict, Iterable, List, Optional
+
+import zigpy.types as t
+import zigpy.zdo.types as zdo_t
+
+
+@dataclass
+class NodeInfo:
+    """Controller Application network Node information."""
+
+    nwk: t.NWK = t.NWK(0xFFFE)
```

Comment:
Is the coordinator's `nwk` ever not `0x0000`?

Reply:
it is not, but NCP could be a router or an end device. But not now.